### PR TITLE
Update Dynamo Projects to use latest IronPython 2.7.9

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Upgrade Dynamo VS Studio solution to reference latest IronPython 2.7.9. As a result of this PR, `Microsoft.VisualStudio.CodeCoverage.Shim.dll` will no longer be part of DynamoCore delivery. I believe there was no real hard dependency on that dll and we included that dll for some unknown reason..

That dll is no longer delivered as part of DynamicRuntimeLanguage 1.2.2 which IronPython 2.7.9 is depending on, see below image
image

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Comment by mjkkirschner** [comment]
@QilongTang if you are worried that integrators need that other dll for patching - we can just supply it as a post build step with a TODO and task to remove it later? - also thanks for jumping on this.

File: test/core/python/IronPythonInfo_TestGraph.dyn [hunk 1]
```diff
@@ -293,7 +293,7 @@
     {
       "ConcreteType": "Dynamo.Graph.Nodes.CodeBlockNodeModel, DynamoCore",
       "NodeType": "CodeBlockNode",
-      "Code": "0;\n5;\n\"2.7.8\";",
+      "Code": "0;\n5;\n\"2.7.9\";",
```
Comment (mjkkirschner) [inline test/core/python/IronPythonInfo_TestGraph.dyn]: hmm, does the test rely on this string in any way?

File: src/Libraries/PythonNodeModels/PythonNode.cs [hunk 2]
```diff
@@ -176,6 +176,7 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
 
         #region SerializeCore/DeserializeCore
 
+        [Obsolete]
```
Comment (aparajit-pratap) [inline src/Libraries/PythonNodeModels/PythonNode.cs]: I must be missing something here. Could you explain this change.

Comment (QilongTang) [reply src/Libraries/PythonNodeModels/PythonNode.cs]: @aparajit-pratap `SerializeCore` is obsoleted in the parent class but not here throwing a warning so I added the tag

File: test/core/python/IronPythonInfo_TestGraph.dyn (hunk 1 above)
Comment (QilongTang) [reply test/core/python/IronPythonInfo_TestGraph.dyn]: Yes

**Comment by QilongTang** [comment]
@aparajit-pratap @mjkkirschner @reddyashish  Since our binary comparison job can only compare nugets and testing on this upgrade did not give my anything obvious yet, I am going to merge this and do a binary comparison so we are sure about what dlls are affected.

**Comment by QilongTang** [comment]
If needed, I can add dependency for `https://www.nuget.org/packages/Microsoft.CodeCoverage/` in a separate PR

**Pull request merged**